ios/Runner/AppDelegate.swift

```diff
@@ -1,13 +1,15 @@
 import UIKit
 import Flutter
 import GoogleMaps
+import Firebase
 
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
   override func application(
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
+      FirebaseApp.configure()
```

Comment:
i dont think this change is needed, `Firebase.initializeApp()` can be done in dart